docs/basics/quick_start_rl/quickstart.md

```diff
@@ -221,31 +220,30 @@ cannot compute the optimal policy, we could simply compare the rewards
 gathered during learning, instead of the regret.
 
 First, we have to record the reward during the fit as this is not done
-automatically. To do this, we can use the
-[WriterWrapper](rlberry.wrappers.writer_utils.WriterWrapper)
-module, or simply the [writer](rlberry.agents.Agent.writer) attribute.
+automatically. To do this, we can use the `writer_extra` optional parameter.
 
 ```python
 class RandomAgent2(RandomAgent):
     name = "RandomAgent2"
 
     def __init__(self, env, **kwargs):
         RandomAgent.__init__(self, env, **kwargs)
-        self.env = WriterWrapper(self.env, self.writer, write_scalar="reward")
 
 
 class UCBVIAgent2(UCBVIAgent):
     name = "UCBVIAgent2"
 
     def __init__(self, env, **kwargs):
         UCBVIAgent.__init__(self, env, **kwargs)
-        self.env = WriterWrapper(self.env, self.writer, write_scalar="reward")
```

Comment:
This is not useful anymore, you can just use `UCBVIAgent` in what follows:
 
```
ucbvi_params["writer_extra"] = "reward"
random_params = {"writer_extra": "reward"}

# Create ExperimentManager for UCBI to fit 10 agents
ucbvi_stats = ExperimentManager(
    UCBVIAgent,
    (env_ctor, env_kwargs),
    fit_budget=50,
    init_kwargs=ucbvi_params,
    n_fit=10,
)
ucbvi_stats.fit()
```
and same for the random agent. This is why I was saying the syntax is simpler, we don't need to redefine a class.